packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -60,17 +72,27 @@
         margin: 2.4rem 0;
         width: 657px;
 
+        @include mobile {
+            width: 100vw;
+            grid-template-columns: unset;
+            justify-content: center;
+        }
+
         &-cell {
             display: block;
             text-align: center;
             width: 158px;
+            @include mobile {
+                width: unset;
+                margin: auto;
+            }
 
             &-header {
                 color: var(--text-less-prominent);
-                font-size: var(--text-size-xs);
                 line-height: 20px;
```

Comment:
You can add all these to the `Text` component and remove the class and `div`s completely.